replica/distributed_loader.cc

```diff
@@ -532,26 +532,28 @@ future<> distributed_loader::populate_keyspace(distributed<replica::database>& d
     auto i = keyspaces.find(ks_name);
     if (i == keyspaces.end()) {
         dblog.warn("Skipping undefined keyspace: {}", ks_name);
-        return make_ready_future<>();
-    } else {
+        co_return;
+    }
+
+    // FIXME: reindent
         dblog.info("Populating Keyspace {}", ks_name);
         auto& ks = i->second;
         auto& column_families = db.local().get_column_families();
 
-        return parallel_for_each(ks.metadata()->cf_meta_data() | boost::adaptors::map_values,
-            [ks_name, ksdir, &ks, &column_families, &db] (schema_ptr s) {
+        for (schema_ptr s : ks.metadata()->cf_meta_data() | boost::adaptors::map_values) {
```

Comment:
But it's not the same as `parallel_for_each`, the parallelizm is lost with this change, isn't it?

Reply:
Yeah, I should've separated that change.
We don't need this level of parallelism.

Reply:
kept parallel_for_each in v3.